### DISCUSSION TITLE
How can I compute the Assortivity or Disassortivity of a node classification dataset?

### DISCUSSION THREAD

**Discussion by jiaruHithub** [pyg-team/pytorch_geometric#3831] [Q&A]
How can I compute the Assortivity or Disassortivity of a node classification dataset (graph)?
I can't find the func. in the library.
Thank you for ur help.

**Comment by rusty1s** [comment]
This should be straightforward to add. Let me know if you have interest in working on this together.

**Reply by jiaruHithub** [reply]
I am honored to contribute to this library. But I don't know if I have enough skills to do the job.

**Reply by rusty1s** [reply]
You mean [this function](https://networkx.org/documentation/stable/reference/algorithms/generated/networkx.algorithms.assortativity.degree_assortativity_coefficient.html#networkx.algorithms.assortativity.degree_assortativity_coefficient), right? As a workaround, can't you simply convert your graph to `networkx` and do the computation there?
